packages/accordion/README.md

```diff
@@ -2,17 +2,14 @@
 
 ## Beskrivelse
 
-Trekkspillister gjør at vi kan gi brukerne mer innhold på liten plass, fordi vi viser informasjonen gradvis. Vi antyder for brukerne at det finnes noe mer, og de kan selv velge å vise eller skjule den ekstra informasjonen.
+Trekkspillister gjør at vi kan gi brukere mer innhold på liten plass ved å vise informasjonen gradvis. De lar brukeren identifiser at det finnes mer innhold som de kan selv velge å vise eller skjule.
 
-Trekkspillister er spesielt nyttig når du designer for mobil, eller i andre tilfeller der du trenger å legge mye informasjon vertikalt.
+Trekkspillisten kan bestå av en eller flere rader. Teksten og vinkeltegnet (chevronen) er trykkbare og viser / skjuler informasjonen. Vi unngår å bruke nestede lister, altså trekkspillister inne i andre trekkspillister. 
```

Comment:
Finnes det et norsk ord for nested slik at vi unngår å forklare hva ordet betyr/innebærer?